### ISSUE TITLE
Fix return of ring_span's copy_popper<T>::operator()

### ISSUE THREAD

**Issue by martinmoene** [WG21-SG14/SG14#91]
in:
```
template <typename T>
T sg14::copy_popper<T>::operator()(T& t)
{
	T old = t;
	t = copy;
	return t;
}
```
shouldn't `return t` be **`return old`**?

**Comment by TBBle** [comment]
Current code matches the proposal, so it would need to be fixed in both places.

**Comment by Quuxplusone** [comment]
Fixed in 5013c6ca13f3c14ed60936852c3a6412c7179f33

**Comment by Quuxplusone** [comment]
Oops, never mind; this is in a different PR of mine now.
